roles/openshift_master_facts/tests/test_default.py

```diff
@@ -0,0 +1,11 @@
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner('.molecule/ansible_inventory').get_hosts('all')
+
+
+def test_hosts_file(File):
+    f = File('/etc/hosts')
+
+    assert f.exists
+    assert f.user == 'root'
+    assert f.group == 'root'
```

Comment:
Yes, they were just placeholders to demonstrate how the testing works rather than real tests.